analyses/molecular-subtyping-MB/05-subtype-mb-shh.R

```diff
@@ -0,0 +1,275 @@
+# Author: Ryan Corbett & Jo Lynne Rokita
+# Function: Script to subtype MB SHH tumors 
+
+# Load packages
+suppressPackageStartupMessages({
+  library(tidyverse)
+  library(data.table)
+})
+
+# Set directory paths
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# Set file paths
+data_dir <- file.path(root_dir, "data")
+analysis_dir <- file.path(root_dir, "analyses", "molecular-subtyping-MB")
+results_dir <- file.path(analysis_dir, "results")
+input_dir <- file.path(analysis_dir, "input")
+plots_dir <- file.path(analysis_dir, "plots")
+
+# set file paths
+hist_file <- file.path(data_dir, "histologies.tsv")
+cn_file <- file.path(data_dir, "consensus_wgs_plus_cnvkit_wxs_plus_freec_tumor_only_autosomes.tsv.gz")
+maf_file <- file.path(data_dir, "snv-consensus-plus-hotspots.maf.tsv.gz")
+tumorOnly_file <- file.path(data_dir, "snv-mutect2-tumor-only-plus-hotspots.maf.tsv.gz")
+expr_file <- file.path(data_dir, "gene-expression-rsem-tpm-collapsed.rds")
+cn_arm_file <- file.path(data_dir, "cnv-consensus-gistic", "broad_values_by_arm.txt")
+germline_file <- file.path(input_dir, "pbta_germline_plp_calls_autogvp_abridged_gnomad_popmax.tsv")
+germline_sv_file <- file.path(input_dir, "pbta_germline_svs.tsv")
+
+# load histologies and subset for mb shh
+hist <- read_tsv(hist_file)
+
+pbta_mb_shh <- hist %>%
+  filter(pathology_diagnosis == "Medulloblastoma",
+         cohort == "PBTA",
+         grepl("SHH", molecular_subtype)) %>%
+  mutate(age_at_diagnosis_years = age_at_diagnosis_days/365.25)
```

Comment:
so then this would need to use `MB_molecular_subtype.tsv` to pull the samples subtyped as MB, SHH, then join with hist base to get ages and mutate